mockwebserver/src/main/java/com/squareup/okhttp/mockwebserver/MockWebServer.java

```diff
@@ -276,22 +278,44 @@ public void start() throws IOException {
   }
 
   /**
-   * Starts the server.
+   * Starts the server on the loopback interface for the given port.
    *
    * @param port the port to listen to, or 0 for any available port. Automated
    *     tests should always use port 0 to avoid flakiness when a specific port
    *     is unavailable.
    */
   public void start(int port) throws IOException {
+    start(InetAddress.getByName("localhost"), port);
+  }
+
+  /**
+   * Starts the server on the given inetaddress and port.
```

Comment:
inetaddress isn't a word. You either want "address" or to wrap it in `{@code .. }`.